### ISSUE TITLE
enhance doccumentation

### ISSUE THREAD

**Issue by nikki-priyaHIT** [nikki-priyaHIT/Project-Exp-Recog#5]

**Comment by astitvaveergarg** [comment]
Hi There, I am a GSSoc-22 Contributor, And Would Like To Contribute In the Project. Please Assign And Give Some Details About Documentation Work To Me.

**Comment by aritra1804** [comment]
Hi There, Aritra here, I would love to contribute to this issue

**Comment by nikki-priyaHIT** [comment]
Hey, @aritra1804 since @astitvaveergarg is working on this issue u can create new one for this project

**Comment by nikki-priyaHIT** [comment]
@astitvaveergarg Are you working on it?

**Comment by aritra1804** [comment]
Hey @nikki-priyaHIT  if Astitva is not working... I can work on it

**Comment by astitvaveergarg** [comment]
@nikki-priyaHIT Yeah, I am Working On Enhancing Documentation, I will Make a PR in max 6 Days.

**Comment by Samhitha1912** [comment]
Hi! I'm Samhitha and GSSOC'22 contributor. I'd like to improve the documentation. Can you pls assign this to me?

**Comment by nikki-priyaHIT** [comment]
@astitvaveergarg give Update please

**Comment by aritra1804** [comment]
@nikki-priyaHIT I can work on the issue, I would complete it by today itself since it is a documentation issue and shouldn't take this much time.
Pls assign it to me

**Comment by aritra1804** [comment]
@nikki-priyaHIT also pls can you also add the gssoc 22 and  the level1,2,3 label in order for points for my other pr else it wont be counted for leaderboard
This is the link of that PR https://github.com/Niketkumardheeryan/ML-CaPsule/pull/288

**Comment by aritra1804** [comment]
@nikki-priyaHIT @nikki-priyaHIT pls put the labels in a correct format. It should be Level1 not Level-1. No special characters allowed and GSSoC22 and not GSSoC' 22. Again no special characters allowed. Pls change it 
This is for my previous PR #30

**Comment by nikki-priyaHIT** [comment]
@aritra1804 Thanks for the correction!

**Comment by nikki-priyaHIT** [comment]
> @nikki-priyaHIT Could you also add the level 1,2,3 label to my previous merged PR else it won't be considered. This is the issue of my previous PR #30

added

**Comment by nikki-priyaHIT** [comment]
@aritra1804 Please update the progress

**Comment by coolblizzarts** [comment]
Hello Payal here, I would like to contribute to this issue. Can you please assign it to me. Regards.

**Comment by nikki-priyaHIT** [comment]
@coolblizzarts  Assigned, Make sure to make PR within 7 days!

**Comment by coolblizzarts** [comment]
> @coolblizzarts Assigned, Make sure to make PR within 7 days!

Hello, I have made the PR. Could you please review it.
